cmd/autocomplete/collaborators_add.go

```diff
@@ -25,16 +27,34 @@ func CollaboratorsAddAutoComplete(c *cli.Context) error {
 		return nil
 	}
 
-	setEmails := make(map[string]bool)
+	ch := make(chan string)
+	var wg sync.WaitGroup
+	wg.Add(len(apps))
 	for _, app := range apps {
-		appCollaborators, err := api.CollaboratorsList(app.Name)
-		if err != nil {
-			return nil
-		}
-		for _, col := range appCollaborators {
-			setEmails[col.Email] = true
-		}
+		go func(app *api.App) {
+			defer wg.Done()
+			appCollaborators, erro := api.CollaboratorsList(app.Name)
+			if erro != nil {
+				err = erro
+				return
+			}
+			for _, col := range appCollaborators {
+				ch <- col.Email
+			}
+		}(app)
 	}
+	if err != nil {
```

Comment:
Checking the error ere has no real meaning, it should be done after the go routines have ended